DQMOffline/Muon/python/muonAnalyzer_cff.py

```diff
@@ -42,3 +42,16 @@
                                   MediumMuonEfficiencyAnalyzer*
                                   TightMuonEfficiencyAnalyzer*                                
                                   muonPFsequence)
+
+from Configuration.StandardSequences.Eras import eras
+for e in [eras.pA_2016]:
+  e.toModify(diMuonHistos, etaBin = cms.int32(175))
+  e.toModify(diMuonHistos, etaBBin = cms.int32(175))
+  e.toModify(diMuonHistos, etaEBin = cms.int32(175))
```

Comment:
@slava77 my fault, sorry: I pointed @goni to the configuration fragment, not to the module initialisation. As you were saying, the file to be edited is [DQMOffline/Muon/python/diMuonHistograms_cfi.py](https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/DQMOffline/Muon/python/diMuonHistograms_cfi.py) with the following lines appended:

````
from Configuration.Eras.Modifier_pA_2016_cff import pA_2016
pA_2016.toModify(diMuonHistos, etaBin = 175, etaBBin = 175,  etaEBin = 175)
````